opbeat/contrib/django/middleware/__init__.py

```diff
@@ -45,50 +60,108 @@ def process_response(self, request, response):
         return response
 
 
-class OpbeatAPMMiddleware(object):
-    # Create a thread local variable to store the session in for logging
-    thread_local = threading.local()
+def get_name_from_middleware(wrapped, instance):
+    name = [type(instance).__name__, wrapped.__name__]
+    if type(instance).__module__:
+        name.insert(0, type(instance).__module__)
```

Comment:
`.__module__` might be `None`, hence the check

https://docs.python.org/2/reference/datamodel.html#index-36